addons/account/views/account_move_views.xml

```diff
@@ -706,20 +706,29 @@
                                 <label for="journal_id"
                                        groups="account.group_account_readonly"
                                        invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
-                                <div name="journal_div" class="d-flex" invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'">
+                                <div name="journal_div"
+                                     class="d-flex"
+                                     groups="account.group_account_readonly"
+                                     invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'">
                                     <field name="journal_id"
-                                        groups="account.group_account_readonly"
-                                        options="{'no_create': True}"
-                                        attrs="{'readonly': [('posted_before', '=', True)]}"/>
-                                    <span class="oe_inline o_form_label mx-3 oe_read_only"
-                                        groups="base.group_multi_currency"> in </span>
-                                    <!-- Bigger margin on the left because in edit mode the external link button covers the text -->
-                                    <span class="oe_inline o_form_label mr-3 ml-5 oe_edit_only"
-                                        groups="base.group_multi_currency"> in </span>
+                                           options="{'no_create': True, 'no_open': True}"
```

Comment:
Was the no-open asked by Benj? In my opinion being able to access the journal from a move isn't a bad thing

Reply:
I'm sorry, it slipped me in copying from v16.0, where I made the PR in the first place.
https://github.com/odoo/odoo/blob/16.0/addons/account/views/account_move_views.xml#L839
Will remove it.